packages/cf-style-container/src/index.js

```diff
@@ -42,6 +42,7 @@ const applyTheme = (ComponentToWrap, ...themes) => {
     }
   }
 
+  ThemedComponent.displayName = `Themed${ComponentToWrap}`;
```

Comment:
Imma add some tests so stuff like this is easier to catch and it's more apparent what we are doing here.